web/src/components/RegisteredForTaxesModal.test.tsx

```diff
@@ -350,6 +363,34 @@ describe("<RegisteredForTaxesModal />", () => {
       expect(currentUserData().profileData.taxId).toEqual("123456789012");
     });
 
+    it("shows the dba name field for dakota nexus when it needs to be inputted", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          legalStructureId: randomPublicFilingLegalStructure(),
+          nexusDbaName: "",
```

Comment:
I think there's a missing test here - is it shown or not shown if the dba field has a value in it?